.stylelintrc

```diff
@@ -0,0 +1,160 @@
+{
+  "plugins": [
+    "stylelint-scss"
+  ],
+  "ignoreFiles": [
```

Comment:
I agree that we should ignore `_generic.normalize.scss` and the test files completely. But can't we circumvent ignoring the other entire files by putting stylelint comments around the rulesets being affected, e.g.:

```scss

/* stylelint-disable selector-list-comma-newline-after */
.selector {
  ...
}
/* stylelint-enable selector-list-comma-newline-after */
```

?